7/Dockerfile.rhel7

```diff
@@ -1,51 +1,50 @@
-FROM rhel7
-MAINTAINER Tom Stellard <tstellar@redhat.com>
+FROM rhscl/s2i-base-rhel7
 
-ENV SUMMARY="Red Hat LLVM Toolset 7 container image" \
-    DESCRIPTION="Platform for building applications using Red Hat LLVM Toolset 7"
+EXPOSE 8080
 
-LABEL com.redhat.component="llvm-toolset-7-docker" \
-      name="devtools/llvm-toolset-7-rhel7" \
-      version="7" \
-      summary="$SUMMARY" \
+ENV LLVM_VERSION="4.0.1" \
+    SUMMARY="Platform for building projects with clang/llvm" \
+    DESCRIPTION="$SUMMARY"
+
+
+LABEL summary="$SUMMARY" \
       description="$DESCRIPTION" \
       io.k8s.description="$DESCRIPTION" \
-      io.k8s.display-name="LLVM Toolset 7"
+      io.k8s.display-name="LLVM 4.0.1" \
```

Comment:
I'd suggest to use only part of the version, that determines the compatibility, so it's not necessary to update this value when newer version is provided by RPM. What I mean by that is -- if it is expected that this image might ever ship newer 4.0.x version, but not 4.1.x, because it might break users' apps builds, then 4.0 might be good value. Or maybe just 4.x if 4.1.x is still fine..

Reply:
I'd also suggest to use the environment variable here in the label, since it's already defined above. just to not duplicate values..